grunt.js

```diff
@@ -314,6 +287,18 @@ module.exports = function(grunt) {
         src: ['build/temp/video.js', 'node_modules/vtt.js/dist/vtt.js'],
         dest: 'build/temp/video.js',
       },
+    },
+    shell: {
+      // Run multiple watch tasks in parallel
+      // Needed so watchify can cache intelligently
+      watchAll: {
+        command: [
+          'grunt connect:dev',
+          'grunt watch',
+          'grunt browserify:watch',
+          'grunt karma:watch'
+        ].join('&')
+      }
```

Comment:
Heh, my bad, should have mentioned that yesterday. What we've got here makes sense, but while we're still in a Grunt world we should probably use grunt-concurrent. Also helps avoid any weirdness around zombie processes that this could potentially introduce.